### ISSUE TITLE
In what ways are you leading the GitHub team that is different from your approach at Heroku?

### ISSUE THREAD

**Issue by saranormous** [jasoncwarner/ama#24]

**Comment by jasoncwarner** [comment]
👋 @saranormous We never met, though big fan. Love what you've been doing at Greylock! 

❤️  the question. Will take some time to respond. 

Something I'll drop in the meantime is I [give a presentation](https://www.slideshare.net/jasoncwarner/architecting-teams-that-scale) on this for VCs and their portfolio companies quite a bit. It's not 100% overlap with your question, but close. 

And one last comment. Much of what we did at Heroku was a few people figuring out Heroku, insides Salesforce, and then scaling everything (teams, systems, processes, gtm, sales, Salesforce wide integration, etc etc). We learned a bunch of good things, had a few missteps and I translated that to GitHub. In some ways I look at Heroku as college football and GitHub as the NFL. I had to translate everything to a bigger stage, higher stakes, faster plays, better team on the other side etc etc. The single biggest difference is that GitHub lacked the overall leadership at the top and had been neglected for a long, long time. Outside of sales and finance, most of GitHub had to be completely rebuilt, particularly product, engineering and the exec room. Heroku had more working structure. 

In short, the stakes were much higher at GitHub, was a complete inflight rebuild, and it came with crazy silicon valley baggage you had to navigate and a lot of historical toxicity to account for even before I was able to get to do the day job of turning around GitHub and making a world class company again.

More later! Love the question.
